src/main/java/racing/domain/Car.java

```diff
@@ -43,4 +42,15 @@ public int compareMeter(Car carB) {
         return 0;
 
     }
+
+    @Override
+    public int compareTo(Car otherCar) {
+        if( this.totalMeter() > otherCar.totalMeter()){
+            return 1;
+        }
+        if( this.totalMeter() < otherCar.totalMeter()){
+            return -1;
+        }
+        return 0;
```

Comment:
다음과 같이 변경해도 좋을 것 같아요 :)
```suggestion
       Integer.compare(this.allMeter, otherCar.allMeter);
```

Reply:
> 다음과 같이 변경해도 좋을 것 같아요 :)
> ```suggestion
>        Integer.compare(this.allMeter, otherCar.allMeter);
> ```

이런 방법도 있었네요 !! 감사합니다